doc/api/tls.markdown

```diff
@@ -249,6 +250,12 @@ automatically set as a listener for the [secureConnection][] event.  The
   - `NPNProtocols`: An array or `Buffer` of possible NPN protocols. (Protocols
     should be ordered by their priority).
 
+  - `ALPNProtocols`: An array or `Buffer` of possible ALPN
+    protocols. (Protocols should be ordered by their priority). When
+    the server receives both NPN and ALPN extensions from the client,
+    ALPN takes precedence over NPN and the server does not send NPN
+    extension to the client.
```

Comment:
Then it should be 'an NPN extension' (or possibly 'the' - reads a little awkward though.)